classes/fullstack/project1/server/src/services/mongodb/models.ts

```diff
@@ -0,0 +1,39 @@
+import mongoose from 'mongoose'
+import { Role, STATUS } from '../../../types'
+
+const adminSchema = new mongoose.Schema(
+  {
+    username: { type: String, unique: true, required: true },
+    password: { type: String, required: true },
+    role: {
+      type: Number,
+      required: true,
+      enum: [Role.ONE, Role.TWO, Role.THREE],
```

Comment:
try and test with 
```
enum:Role
enum:StATUS
```

Reply:
I have added parseInt to address inconsistency between mongodb type (number) and Role interface (string):

```
 role: {
      type: Number,
      required: true,
      enum: [parseInt(Role.ONE), parseInt(Role.TWO), parseInt(Role.THREE)],
    },
```